designs/simone/config/index.js

```diff
@@ -1,6 +1,24 @@
 import { version } from '../package.json'
 import { config as simonConfig } from '@freesewing/simon'
 
+// Add the Bust-aligned Buttons option into the "Style->Closure" submenu.
+// - Because the closure submenu is an object in the style array,
+//   we cannot simply replace/overwrite it by adding a new closure
+//   submenu. Doing so would just create a duplicate "Closure"
+//   submenu.
+// - Instead, we would need to remove the existing closure submenu
+//   object from simonConfig.optionGroups.style first, before adding
+//   the new closure submenu. (Doing it this way would also have the
+//   side effect of changing the position of the "Closure" submenu
+//   within the main"Style" menu.)
+// - Alternately, we can simply modify the existing closure submenu
+//   object in place, and that is what we are doing here: injecting the
+//   new 'bustAlignedButtons' option directly into the closure submenu
+//   array. This seems to be the least disruptive method.
+const closureindex = simonConfig.optionGroups.style.findIndex((item) =>
+    typeof item === 'object' && "closure" in item)
+simonConfig.optionGroups.style[closureindex].closure.push('bustAlignedButtons')
```

Comment:
Nicely done. FYI, we're going to change this in v3 to make it easier to handle this.
We'll only accept a top-level key and an array of options. For nested structure, you would use a `:` in they key.

So here you would have just added:
```js
"style:closure": [ "bustAlignedButtons"]
```
And that would then get merged into the closure group.

Still on the fence about using `:` or `.` as a separator, but this is the idea.